sinabs/conversion.py

```diff
@@ -32,11 +38,21 @@ def replace_module_(model: nn.Sequential, source_class: type, mapper_fn: Callabl
         mapper_fn: A callable that takes as argument the layer to replace and returns the new object.
 
     Returns:
-        None. Model is modified in-place.
+        Modified model.
```

Comment:
this is a bit confusing I think because this function doesn't return anything. What if we just delete Returns and add a `-> None` in the function signature?